packages/common/src/events/_internal/EventUiSchemaEdit.ts

```diff
@@ -182,6 +182,29 @@ export const buildUiSchema = async (
               ],
             },
           },
+          {
+            labelKey: `${i18nScope}.fields.timeZone.label`,
+            scope: "/properties/timeZone",
+            type: "Control",
+            rule: {
```

Comment:
I _believe_ we'd want to show the `timeZone` control regardless of whether the event is considered all-day or not.